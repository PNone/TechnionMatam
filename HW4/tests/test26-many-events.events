PotionsMerchant
PotionsMerchant PotionsMerchant PotionsMerchant PotionsMerchant PotionsMerchant
PotionsMerchant PotionsMerchant
PotionsMerchant PotionsMerchant PotionsMerchant PotionsMerchant
PotionsMerchant
PotionsMerchant PotionsMerchant
PotionsMerchant PotionsMerchant PotionsMerchant PotionsMerchant
PotionsMerchant
PotionsMerchant
PotionsMerchant
PotionsMerchant PotionsMerchant PotionsMerchant PotionsMerchant
PotionsMerchant
PotionsMerchant
PotionsMerchant
PotionsMerchant SolarEclipse SolarEclipse SolarEclipse SolarEclipse
Snail
Snail
Snail
Slime  Slime Slime Slime     Slime          Slime
Balrog Pack 3 Snail Snail Snail