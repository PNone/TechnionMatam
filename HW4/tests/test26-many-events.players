Braa Warrior Responsible
Regev Magician RiskTaking
Adi Archer Responsible
Ben Magician Responsible
Sean Archer RiskTaking