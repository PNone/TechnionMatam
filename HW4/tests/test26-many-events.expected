Game is about to start!
These are the players:

1. Braa, Warrior with Responsible character (level 1, force 5)
2. Regev, Magician with RiskTaking character (level 1, force 5)
3. Adi, Archer with Responsible character (level 1, force 5)
4. Ben, Magician with Responsible character (level 1, force 5)
5. Sean, Archer with RiskTaking character (level 1, force 5)

----------------------------------------

New round starts!

Turn: 1
Player: Braa, Warrior with Responsible character (level 1, force 5)
Event: PotionsMerchant
Braa bought 0 potions!

Turn: 2
Player: Regev, Magician with RiskTaking character (level 1, force 5)
Event: PotionsMerchant
Regev bought 0 potions!

Turn: 3
Player: Adi, Archer with Responsible character (level 1, force 5)
Event: PotionsMerchant
Adi bought 0 potions!

Turn: 4
Player: Ben, Magician with Responsible character (level 1, force 5)
Event: PotionsMerchant
Ben bought 0 potions!

Turn: 5
Player: Sean, Archer with RiskTaking character (level 1, force 5)
Event: PotionsMerchant
Sean bought 0 potions!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Adi             1         5     100   20
2          Sean            1         5     100   20
3          Ben             1         5     100   10
4          Braa            1         5     150   10
5          Regev           1         5     100   10

----------------------------------------

New round starts!

Turn: 6
Player: Braa, Warrior with Responsible character (level 1, force 5)
Event: PotionsMerchant
Braa bought 0 potions!

Turn: 7
Player: Regev, Magician with RiskTaking character (level 1, force 5)
Event: PotionsMerchant
Regev bought 0 potions!

Turn: 8
Player: Adi, Archer with Responsible character (level 1, force 5)
Event: PotionsMerchant
Adi bought 0 potions!

Turn: 9
Player: Ben, Magician with Responsible character (level 1, force 5)
Event: PotionsMerchant
Ben bought 0 potions!

Turn: 10
Player: Sean, Archer with RiskTaking character (level 1, force 5)
Event: PotionsMerchant
Sean bought 0 potions!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Adi             1         5     100   20
2          Sean            1         5     100   20
3          Ben             1         5     100   10
4          Braa            1         5     150   10
5          Regev           1         5     100   10

----------------------------------------

New round starts!

Turn: 11
Player: Braa, Warrior with Responsible character (level 1, force 5)
Event: PotionsMerchant
Braa bought 0 potions!

Turn: 12
Player: Regev, Magician with RiskTaking character (level 1, force 5)
Event: PotionsMerchant
Regev bought 0 potions!

Turn: 13
Player: Adi, Archer with Responsible character (level 1, force 5)
Event: PotionsMerchant
Adi bought 0 potions!

Turn: 14
Player: Ben, Magician with Responsible character (level 1, force 5)
Event: PotionsMerchant
Ben bought 0 potions!

Turn: 15
Player: Sean, Archer with RiskTaking character (level 1, force 5)
Event: PotionsMerchant
Sean bought 0 potions!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Adi             1         5     100   20
2          Sean            1         5     100   20
3          Ben             1         5     100   10
4          Braa            1         5     150   10
5          Regev           1         5     100   10

----------------------------------------

New round starts!

Turn: 16
Player: Braa, Warrior with Responsible character (level 1, force 5)
Event: PotionsMerchant
Braa bought 0 potions!

Turn: 17
Player: Regev, Magician with RiskTaking character (level 1, force 5)
Event: PotionsMerchant
Regev bought 0 potions!

Turn: 18
Player: Adi, Archer with Responsible character (level 1, force 5)
Event: PotionsMerchant
Adi bought 0 potions!

Turn: 19
Player: Ben, Magician with Responsible character (level 1, force 5)
Event: PotionsMerchant
Ben bought 0 potions!

Turn: 20
Player: Sean, Archer with RiskTaking character (level 1, force 5)
Event: PotionsMerchant
Sean bought 0 potions!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Adi             1         5     100   20
2          Sean            1         5     100   20
3          Ben             1         5     100   10
4          Braa            1         5     150   10
5          Regev           1         5     100   10

----------------------------------------

New round starts!

Turn: 21
Player: Braa, Warrior with Responsible character (level 1, force 5)
Event: PotionsMerchant
Braa bought 0 potions!

Turn: 22
Player: Regev, Magician with RiskTaking character (level 1, force 5)
Event: PotionsMerchant
Regev bought 0 potions!

Turn: 23
Player: Adi, Archer with Responsible character (level 1, force 5)
Event: PotionsMerchant
Adi bought 0 potions!

Turn: 24
Player: Ben, Magician with Responsible character (level 1, force 5)
Event: PotionsMerchant
Ben bought 0 potions!

Turn: 25
Player: Sean, Archer with RiskTaking character (level 1, force 5)
Event: PotionsMerchant
Sean bought 0 potions!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Adi             1         5     100   20
2          Sean            1         5     100   20
3          Ben             1         5     100   10
4          Braa            1         5     150   10
5          Regev           1         5     100   10

----------------------------------------

New round starts!

Turn: 26
Player: Braa, Warrior with Responsible character (level 1, force 5)
Event: PotionsMerchant
Braa bought 0 potions!

Turn: 27
Player: Regev, Magician with RiskTaking character (level 1, force 5)
Event: PotionsMerchant
Regev bought 0 potions!

Turn: 28
Player: Adi, Archer with Responsible character (level 1, force 5)
Event: PotionsMerchant
Adi bought 0 potions!

Turn: 29
Player: Ben, Magician with Responsible character (level 1, force 5)
Event: PotionsMerchant
Ben bought 0 potions!

Turn: 30
Player: Sean, Archer with RiskTaking character (level 1, force 5)
Event: PotionsMerchant
Sean bought 0 potions!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Adi             1         5     100   20
2          Sean            1         5     100   20
3          Ben             1         5     100   10
4          Braa            1         5     150   10
5          Regev           1         5     100   10

----------------------------------------

New round starts!

Turn: 31
Player: Braa, Warrior with Responsible character (level 1, force 5)
Event: SolarEclipse
Braa was affected by a solar eclipse! their force has changed by -1!

Turn: 32
Player: Regev, Magician with RiskTaking character (level 1, force 5)
Event: SolarEclipse
Regev was affected by a solar eclipse! their force has changed by 1!

Turn: 33
Player: Adi, Archer with Responsible character (level 1, force 5)
Event: SolarEclipse
Adi was affected by a solar eclipse! their force has changed by -1!

Turn: 34
Player: Ben, Magician with Responsible character (level 1, force 5)
Event: SolarEclipse
Ben was affected by a solar eclipse! their force has changed by 1!

Turn: 35
Player: Sean, Archer with RiskTaking character (level 1, force 5)
Event: Snail (power 5, loot 2, damage 10)
Sean won the encounter, gained 2 coins and leveled up!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Sean            2         5     100   22
2          Adi             1         4     100   20
3          Ben             1         6     100   10
4          Braa            1         4     150   10
5          Regev           1         6     100   10

----------------------------------------

New round starts!

Turn: 36
Player: Braa, Warrior with Responsible character (level 1, force 4)
Event: Snail (power 5, loot 2, damage 10)
Braa won the encounter, gained 2 coins and leveled up!

Turn: 37
Player: Regev, Magician with RiskTaking character (level 1, force 6)
Event: Snail (power 5, loot 2, damage 10)
Regev won the encounter, gained 2 coins and leveled up!

Turn: 38
Player: Adi, Archer with Responsible character (level 1, force 4)
Event: Slime (power 12, loot 5, damage 25)
Adi lost the encounter and took 25 damage!

Turn: 39
Player: Ben, Magician with Responsible character (level 1, force 6)
Event: Slime (power 12, loot 5, damage 25)
Ben lost the encounter and took 25 damage!

Turn: 40
Player: Sean, Archer with RiskTaking character (level 2, force 5)
Event: Slime (power 12, loot 5, damage 25)
Sean lost the encounter and took 25 damage!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Sean            2         5     75    22
2          Braa            2         4     140   12
3          Regev           2         6     100   12
4          Adi             1         4     75    20
5          Ben             1         6     75    10

----------------------------------------

New round starts!

Turn: 41
Player: Braa, Warrior with Responsible character (level 2, force 4)
Event: Slime (power 12, loot 5, damage 25)
Braa lost the encounter and took 25 damage!

Turn: 42
Player: Regev, Magician with RiskTaking character (level 2, force 6)
Event: Slime (power 12, loot 5, damage 25)
Regev lost the encounter and took 25 damage!

Turn: 43
Player: Adi, Archer with Responsible character (level 1, force 4)
Event: Slime (power 12, loot 5, damage 25)
Adi lost the encounter and took 25 damage!

Turn: 44
Player: Ben, Magician with Responsible character (level 1, force 6)
Event: Balrog (power 15, loot 100, damage 9001)
Ben lost the encounter and took 9001 damage!

Turn: 45
Player: Sean, Archer with RiskTaking character (level 2, force 5)
Event: Pack of 3 members (power 15, loot 6, damage 30)
Sean lost the encounter and took 30 damage!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Sean            2         5     45    22
2          Braa            2         4     105   12
3          Regev           2         6     75    12
4          Adi             1         4     50    20
5          Ben             1         6     0     10

----------------------------------------

New round starts!

Turn: 46
Player: Braa, Warrior with Responsible character (level 2, force 4)
Event: PotionsMerchant
Braa bought 2 potions!

Turn: 47
Player: Regev, Magician with RiskTaking character (level 2, force 6)
Event: PotionsMerchant
Regev bought 0 potions!

Turn: 48
Player: Adi, Archer with Responsible character (level 1, force 4)
Event: PotionsMerchant
Adi bought 4 potions!

Turn: 49
Player: Sean, Archer with RiskTaking character (level 2, force 5)
Event: PotionsMerchant
Sean bought 1 potions!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Sean            2         5     55    17
2          Regev           2         6     75    12
3          Braa            2         4     125   2
4          Ben             1         6     0     10
5          Adi             1         4     90    0

----------------------------------------

New round starts!

Turn: 50
Player: Braa, Warrior with Responsible character (level 2, force 4)
Event: PotionsMerchant
Braa bought 0 potions!

Turn: 51
Player: Regev, Magician with RiskTaking character (level 2, force 6)
Event: PotionsMerchant
Regev bought 0 potions!

Turn: 52
Player: Adi, Archer with Responsible character (level 1, force 4)
Event: PotionsMerchant
Adi bought 0 potions!

Turn: 53
Player: Sean, Archer with RiskTaking character (level 2, force 5)
Event: PotionsMerchant
Sean bought 0 potions!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Sean            2         5     55    17
2          Regev           2         6     75    12
3          Braa            2         4     125   2
4          Ben             1         6     0     10
5          Adi             1         4     90    0

----------------------------------------

New round starts!

Turn: 54
Player: Braa, Warrior with Responsible character (level 2, force 4)
Event: PotionsMerchant
Braa bought 0 potions!

Turn: 55
Player: Regev, Magician with RiskTaking character (level 2, force 6)
Event: PotionsMerchant
Regev bought 0 potions!

Turn: 56
Player: Adi, Archer with Responsible character (level 1, force 4)
Event: PotionsMerchant
Adi bought 0 potions!

Turn: 57
Player: Sean, Archer with RiskTaking character (level 2, force 5)
Event: PotionsMerchant
Sean bought 0 potions!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Sean            2         5     55    17
2          Regev           2         6     75    12
3          Braa            2         4     125   2
4          Ben             1         6     0     10
5          Adi             1         4     90    0

----------------------------------------

New round starts!

Turn: 58
Player: Braa, Warrior with Responsible character (level 2, force 4)
Event: PotionsMerchant
Braa bought 0 potions!

Turn: 59
Player: Regev, Magician with RiskTaking character (level 2, force 6)
Event: PotionsMerchant
Regev bought 0 potions!

Turn: 60
Player: Adi, Archer with Responsible character (level 1, force 4)
Event: PotionsMerchant
Adi bought 0 potions!

Turn: 61
Player: Sean, Archer with RiskTaking character (level 2, force 5)
Event: PotionsMerchant
Sean bought 0 potions!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Sean            2         5     55    17
2          Regev           2         6     75    12
3          Braa            2         4     125   2
4          Ben             1         6     0     10
5          Adi             1         4     90    0

----------------------------------------

New round starts!

Turn: 62
Player: Braa, Warrior with Responsible character (level 2, force 4)
Event: PotionsMerchant
Braa bought 0 potions!

Turn: 63
Player: Regev, Magician with RiskTaking character (level 2, force 6)
Event: PotionsMerchant
Regev bought 0 potions!

Turn: 64
Player: Adi, Archer with Responsible character (level 1, force 4)
Event: PotionsMerchant
Adi bought 0 potions!

Turn: 65
Player: Sean, Archer with RiskTaking character (level 2, force 5)
Event: PotionsMerchant
Sean bought 0 potions!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Sean            2         5     55    17
2          Regev           2         6     75    12
3          Braa            2         4     125   2
4          Ben             1         6     0     10
5          Adi             1         4     90    0

----------------------------------------

New round starts!

Turn: 66
Player: Braa, Warrior with Responsible character (level 2, force 4)
Event: PotionsMerchant
Braa bought 0 potions!

Turn: 67
Player: Regev, Magician with RiskTaking character (level 2, force 6)
Event: PotionsMerchant
Regev bought 0 potions!

Turn: 68
Player: Adi, Archer with Responsible character (level 1, force 4)
Event: PotionsMerchant
Adi bought 0 potions!

Turn: 69
Player: Sean, Archer with RiskTaking character (level 2, force 5)
Event: PotionsMerchant
Sean bought 0 potions!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Sean            2         5     55    17
2          Regev           2         6     75    12
3          Braa            2         4     125   2
4          Ben             1         6     0     10
5          Adi             1         4     90    0

----------------------------------------

New round starts!

Turn: 70
Player: Braa, Warrior with Responsible character (level 2, force 4)
Event: PotionsMerchant
Braa bought 0 potions!

Turn: 71
Player: Regev, Magician with RiskTaking character (level 2, force 6)
Event: PotionsMerchant
Regev bought 0 potions!

Turn: 72
Player: Adi, Archer with Responsible character (level 1, force 4)
Event: PotionsMerchant
Adi bought 0 potions!

Turn: 73
Player: Sean, Archer with RiskTaking character (level 2, force 5)
Event: PotionsMerchant
Sean bought 0 potions!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Sean            2         5     55    17
2          Regev           2         6     75    12
3          Braa            2         4     125   2
4          Ben             1         6     0     10
5          Adi             1         4     90    0

----------------------------------------

New round starts!

Turn: 74
Player: Braa, Warrior with Responsible character (level 2, force 4)
Event: PotionsMerchant
Braa bought 0 potions!

Turn: 75
Player: Regev, Magician with RiskTaking character (level 2, force 6)
Event: PotionsMerchant
Regev bought 0 potions!

Turn: 76
Player: Adi, Archer with Responsible character (level 1, force 4)
Event: SolarEclipse
Adi was affected by a solar eclipse! their force has changed by -1!

Turn: 77
Player: Sean, Archer with RiskTaking character (level 2, force 5)
Event: SolarEclipse
Sean was affected by a solar eclipse! their force has changed by -1!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Sean            2         4     55    17
2          Regev           2         6     75    12
3          Braa            2         4     125   2
4          Ben             1         6     0     10
5          Adi             1         3     90    0

----------------------------------------

New round starts!

Turn: 78
Player: Braa, Warrior with Responsible character (level 2, force 4)
Event: SolarEclipse
Braa was affected by a solar eclipse! their force has changed by -1!

Turn: 79
Player: Regev, Magician with RiskTaking character (level 2, force 6)
Event: SolarEclipse
Regev was affected by a solar eclipse! their force has changed by 1!

Turn: 80
Player: Adi, Archer with Responsible character (level 1, force 3)
Event: Snail (power 5, loot 2, damage 10)
Adi lost the encounter and took 10 damage!

Turn: 81
Player: Sean, Archer with RiskTaking character (level 2, force 4)
Event: Snail (power 5, loot 2, damage 10)
Sean won the encounter, gained 2 coins and leveled up!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Sean            3         4     55    19
2          Regev           2         7     75    12
3          Braa            2         3     125   2
4          Ben             1         6     0     10
5          Adi             1         3     80    0

----------------------------------------

New round starts!

Turn: 82
Player: Braa, Warrior with Responsible character (level 2, force 3)
Event: Snail (power 5, loot 2, damage 10)
Braa won the encounter, gained 2 coins and leveled up!

Turn: 83
Player: Regev, Magician with RiskTaking character (level 2, force 7)
Event: Slime (power 12, loot 5, damage 25)
Regev lost the encounter and took 25 damage!

Turn: 84
Player: Adi, Archer with Responsible character (level 1, force 3)
Event: Slime (power 12, loot 5, damage 25)
Adi lost the encounter and took 25 damage!

Turn: 85
Player: Sean, Archer with RiskTaking character (level 3, force 4)
Event: Slime (power 12, loot 5, damage 25)
Sean lost the encounter and took 25 damage!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Sean            3         4     30    19
2          Braa            3         3     115   4
3          Regev           2         7     50    12
4          Ben             1         6     0     10
5          Adi             1         3     55    0

----------------------------------------

New round starts!

Turn: 86
Player: Braa, Warrior with Responsible character (level 3, force 3)
Event: Slime (power 12, loot 5, damage 25)
Braa lost the encounter and took 25 damage!

Turn: 87
Player: Regev, Magician with RiskTaking character (level 2, force 7)
Event: Slime (power 12, loot 5, damage 25)
Regev lost the encounter and took 25 damage!

Turn: 88
Player: Adi, Archer with Responsible character (level 1, force 3)
Event: Slime (power 12, loot 5, damage 25)
Adi lost the encounter and took 25 damage!

Turn: 89
Player: Sean, Archer with RiskTaking character (level 3, force 4)
Event: Balrog (power 17, loot 100, damage 9001)
Sean lost the encounter and took 9001 damage!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Sean            3         4     0     19
2          Braa            3         3     80    4
3          Regev           2         7     25    12
4          Ben             1         6     0     10
5          Adi             1         3     30    0

----------------------------------------

New round starts!

Turn: 90
Player: Braa, Warrior with Responsible character (level 3, force 3)
Event: Pack of 3 members (power 15, loot 6, damage 30)
Braa lost the encounter and took 30 damage!

Turn: 91
Player: Regev, Magician with RiskTaking character (level 2, force 7)
Event: PotionsMerchant
Regev bought 1 potions!

Turn: 92
Player: Adi, Archer with Responsible character (level 1, force 3)
Event: PotionsMerchant
Adi bought 0 potions!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Sean            3         4     0     19
2          Braa            3         3     40    4
3          Regev           2         7     35    7
4          Ben             1         6     0     10
5          Adi             1         3     30    0

----------------------------------------

New round starts!

Turn: 93
Player: Braa, Warrior with Responsible character (level 3, force 3)
Event: PotionsMerchant
Braa bought 0 potions!

Turn: 94
Player: Regev, Magician with RiskTaking character (level 2, force 7)
Event: PotionsMerchant
Regev bought 1 potions!

Turn: 95
Player: Adi, Archer with Responsible character (level 1, force 3)
Event: PotionsMerchant
Adi bought 0 potions!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Sean            3         4     0     19
2          Braa            3         3     40    4
3          Regev           2         7     45    2
4          Ben             1         6     0     10
5          Adi             1         3     30    0

----------------------------------------

New round starts!

Turn: 96
Player: Braa, Warrior with Responsible character (level 3, force 3)
Event: PotionsMerchant
Braa bought 0 potions!

Turn: 97
Player: Regev, Magician with RiskTaking character (level 2, force 7)
Event: PotionsMerchant
Regev bought 0 potions!

Turn: 98
Player: Adi, Archer with Responsible character (level 1, force 3)
Event: PotionsMerchant
Adi bought 0 potions!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Sean            3         4     0     19
2          Braa            3         3     40    4
3          Regev           2         7     45    2
4          Ben             1         6     0     10
5          Adi             1         3     30    0

----------------------------------------

New round starts!

Turn: 99
Player: Braa, Warrior with Responsible character (level 3, force 3)
Event: PotionsMerchant
Braa bought 0 potions!

Turn: 100
Player: Regev, Magician with RiskTaking character (level 2, force 7)
Event: PotionsMerchant
Regev bought 0 potions!

Turn: 101
Player: Adi, Archer with Responsible character (level 1, force 3)
Event: PotionsMerchant
Adi bought 0 potions!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Sean            3         4     0     19
2          Braa            3         3     40    4
3          Regev           2         7     45    2
4          Ben             1         6     0     10
5          Adi             1         3     30    0

----------------------------------------

New round starts!

Turn: 102
Player: Braa, Warrior with Responsible character (level 3, force 3)
Event: PotionsMerchant
Braa bought 0 potions!

Turn: 103
Player: Regev, Magician with RiskTaking character (level 2, force 7)
Event: PotionsMerchant
Regev bought 0 potions!

Turn: 104
Player: Adi, Archer with Responsible character (level 1, force 3)
Event: PotionsMerchant
Adi bought 0 potions!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Sean            3         4     0     19
2          Braa            3         3     40    4
3          Regev           2         7     45    2
4          Ben             1         6     0     10
5          Adi             1         3     30    0

----------------------------------------

New round starts!

Turn: 105
Player: Braa, Warrior with Responsible character (level 3, force 3)
Event: PotionsMerchant
Braa bought 0 potions!

Turn: 106
Player: Regev, Magician with RiskTaking character (level 2, force 7)
Event: PotionsMerchant
Regev bought 0 potions!

Turn: 107
Player: Adi, Archer with Responsible character (level 1, force 3)
Event: PotionsMerchant
Adi bought 0 potions!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Sean            3         4     0     19
2          Braa            3         3     40    4
3          Regev           2         7     45    2
4          Ben             1         6     0     10
5          Adi             1         3     30    0

----------------------------------------

New round starts!

Turn: 108
Player: Braa, Warrior with Responsible character (level 3, force 3)
Event: PotionsMerchant
Braa bought 0 potions!

Turn: 109
Player: Regev, Magician with RiskTaking character (level 2, force 7)
Event: PotionsMerchant
Regev bought 0 potions!

Turn: 110
Player: Adi, Archer with Responsible character (level 1, force 3)
Event: PotionsMerchant
Adi bought 0 potions!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Sean            3         4     0     19
2          Braa            3         3     40    4
3          Regev           2         7     45    2
4          Ben             1         6     0     10
5          Adi             1         3     30    0

----------------------------------------

New round starts!

Turn: 111
Player: Braa, Warrior with Responsible character (level 3, force 3)
Event: PotionsMerchant
Braa bought 0 potions!

Turn: 112
Player: Regev, Magician with RiskTaking character (level 2, force 7)
Event: PotionsMerchant
Regev bought 0 potions!

Turn: 113
Player: Adi, Archer with Responsible character (level 1, force 3)
Event: PotionsMerchant
Adi bought 0 potions!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Sean            3         4     0     19
2          Braa            3         3     40    4
3          Regev           2         7     45    2
4          Ben             1         6     0     10
5          Adi             1         3     30    0

----------------------------------------

New round starts!

Turn: 114
Player: Braa, Warrior with Responsible character (level 3, force 3)
Event: PotionsMerchant
Braa bought 0 potions!

Turn: 115
Player: Regev, Magician with RiskTaking character (level 2, force 7)
Event: PotionsMerchant
Regev bought 0 potions!

Turn: 116
Player: Adi, Archer with Responsible character (level 1, force 3)
Event: PotionsMerchant
Adi bought 0 potions!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Sean            3         4     0     19
2          Braa            3         3     40    4
3          Regev           2         7     45    2
4          Ben             1         6     0     10
5          Adi             1         3     30    0

----------------------------------------

New round starts!

Turn: 117
Player: Braa, Warrior with Responsible character (level 3, force 3)
Event: PotionsMerchant
Braa bought 0 potions!

Turn: 118
Player: Regev, Magician with RiskTaking character (level 2, force 7)
Event: PotionsMerchant
Regev bought 0 potions!

Turn: 119
Player: Adi, Archer with Responsible character (level 1, force 3)
Event: PotionsMerchant
Adi bought 0 potions!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Sean            3         4     0     19
2          Braa            3         3     40    4
3          Regev           2         7     45    2
4          Ben             1         6     0     10
5          Adi             1         3     30    0

----------------------------------------

New round starts!

Turn: 120
Player: Braa, Warrior with Responsible character (level 3, force 3)
Event: PotionsMerchant
Braa bought 0 potions!

Turn: 121
Player: Regev, Magician with RiskTaking character (level 2, force 7)
Event: SolarEclipse
Regev was affected by a solar eclipse! their force has changed by 1!

Turn: 122
Player: Adi, Archer with Responsible character (level 1, force 3)
Event: SolarEclipse
Adi was affected by a solar eclipse! their force has changed by -1!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Sean            3         4     0     19
2          Braa            3         3     40    4
3          Regev           2         8     45    2
4          Ben             1         6     0     10
5          Adi             1         2     30    0

----------------------------------------

New round starts!

Turn: 123
Player: Braa, Warrior with Responsible character (level 3, force 3)
Event: SolarEclipse
Braa was affected by a solar eclipse! their force has changed by -1!

Turn: 124
Player: Regev, Magician with RiskTaking character (level 2, force 8)
Event: SolarEclipse
Regev was affected by a solar eclipse! their force has changed by 1!

Turn: 125
Player: Adi, Archer with Responsible character (level 1, force 2)
Event: Snail (power 5, loot 2, damage 10)
Adi lost the encounter and took 10 damage!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Sean            3         4     0     19
2          Braa            3         2     40    4
3          Regev           2         9     45    2
4          Ben             1         6     0     10
5          Adi             1         2     20    0

----------------------------------------

New round starts!

Turn: 126
Player: Braa, Warrior with Responsible character (level 3, force 2)
Event: Snail (power 5, loot 2, damage 10)
Braa won the encounter, gained 2 coins and leveled up!

Turn: 127
Player: Regev, Magician with RiskTaking character (level 2, force 9)
Event: Snail (power 5, loot 2, damage 10)
Regev won the encounter, gained 2 coins and leveled up!

Turn: 128
Player: Adi, Archer with Responsible character (level 1, force 2)
Event: Slime (power 12, loot 5, damage 25)
Adi lost the encounter and took 25 damage!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Braa            4         2     30    6
2          Sean            3         4     0     19
3          Regev           3         9     45    4
4          Ben             1         6     0     10
5          Adi             1         2     0     0

----------------------------------------

New round starts!

Turn: 129
Player: Braa, Warrior with Responsible character (level 4, force 2)
Event: Slime (power 12, loot 5, damage 25)
Braa lost the encounter and took 25 damage!

Turn: 130
Player: Regev, Magician with RiskTaking character (level 3, force 9)
Event: Slime (power 12, loot 5, damage 25)
Regev lost the encounter and took 25 damage!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Braa            4         2     0     6
2          Sean            3         4     0     19
3          Regev           3         9     20    4
4          Ben             1         6     0     10
5          Adi             1         2     0     0

----------------------------------------

New round starts!

Turn: 131
Player: Regev, Magician with RiskTaking character (level 3, force 9)
Event: Slime (power 12, loot 5, damage 25)
Regev lost the encounter and took 25 damage!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Braa            4         2     0     6
2          Sean            3         4     0     19
3          Regev           3         9     0     4
4          Ben             1         6     0     10
5          Adi             1         2     0     0

----------------------------------------

Game is over!
There are no winners in this game.
Better luck next time!
